src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,31 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
+    {
+        if (minValidations < 1)
+            minValidations = 1;
+
+        std::vector <std::uint32_t> times;
+
```

Comment:
@scottschurr You gotta be careful with what you pass for the reservation. We might reserve based on the number of validators in the UNL rather than the number of validations that we have.